packages/import-bundle/NEWS.md

```diff
@@ -1,5 +1,16 @@
 User-visible changes to `@endo/import-bundle`:
 
+# Next release
+
+Experimental:
+
+- Adds a `__native__: true` option that indicates that the application will
+  fall through to the native implementation of Compartment, currently only
+  available on XS, which lacks support for precompiled module sources (as exist
+  in many archived applications, particularly Agoric smart contract bundles)
+  and instead supports loading modules from original sources (which is not
+  possible at runtime on XS).
```

Comment:
An alternative design that I am interested in considering is to instead create a new `moduleFormat` called `endoNativeZipBase64` that implies this `__native__` flag. Whether it can be overridden is worth consideration. This would imply a change in `bundle-source` such that the existing `--no-transforms` or a new `--native` flags would switch the `moduleFormat`. We can create bundles with `-T,--no-transforms` today that cannot be imported without this `__native__` flag on XS, but once this PR lands, can be imported on Node.js.

Reply:
I don't understand

> loading modules from original sources (which is not possible at runtime on XS)

What about this is not possible on XS?

Reply:
SES on XS, both before this change and after this change (but without the `__native__` option set), cannot load a module from original sources without the assistance of the _optional_ `ModuleSource` shim.

In any case, this prose needs some work, but I’ve also moved this commit out of this PR since I intend to investigate the alternate design using an `"endoNativeZipBase64"` format. The MetaMask folks reminded me that the web will still benefit from pre-compiled bundles, so those should remain the default and we should refrain from changing the default even with a major version bump on `bundle-source`.